src/SfdxProjectBuilder.groovy

```diff
@@ -90,6 +90,31 @@ class SfdxProjectBuilder implements Serializable {
   // the parsed contents of the SFDX project's configuration
   private def sfdxPackage
 
+  // dynamic property scaffold
+  private String packagingAlias() {
+    if ( _.env.SFDX_PKG_HUB_USERNAME?.trim() && _.env.SFDX_PKG_HUB_HOST?.trim() ) {
+      return _.env.SFDX_PKG_HUB_ALIAS
+    } else {
+      return _.env.SFDX_DEV_HUB_ALIAS
+    }
+  }
+
+  private String packagingHost() {
```

Comment:
@TheRickParker, I am not seeing this method used anywhere.  Am I missing something?